src/main.js

```diff
@@ -94,13 +115,21 @@ const parseVarFromType = envSpecAsArray => {
   }));
 };
 
-//function that creates the render label for the output outputHTML
+/** @function renderLabelForEntry
+ * @desc Creates the render label for the {@link outputHTML} function.
```

Comment:
> Creates the render label for the

We should prefer: *Creates the render for the*

"Render" is a verb we use to express the transformation of code into something visible by a human (e.g. HTML code is being rendered into an actual button).